juspay/hyperswitch-control-center
# fix: single status view setting to all

## Type of Change




## Description

- single status filter for which view did not exist was setting to All - fixed the issue with this PR

![image](https://github.com/user-attachments/assets/2d8005f4-5770-48c7-b66f-51f685829df1)

## Motivation and Context



## How did you test it?

1. go to orders/refunds/disputes
2. apply status filter
3. choose a single value that does not exist in views above
4. no view should be shown as selected

## Where to test it?


## Checklist
